.travis.yml

```diff
@@ -68,7 +68,8 @@ matrix:
         - pip install -r mbed-os/requirements.txt
       script:
         # Build TF-M and all tests
-        - python3 build_tfm.py -t GNUARM -m ${TARGET_NAME}
```

Comment:
I think we still build the `CoreIPC.cmake` config. pass -c as an argument and make it as a new matrix job.

Reply:
The eventual objective is switch back to `test_psa_target.py`, so for now I'd prefer not extending the matrix which might not be needed later. Also, the `mbed compile` won't work for ConfigCoreIPC.cmake because it lacks test symbols.
For a temporary sanity check, shall I add
```
python3 build_tfm.py -t GNUARM -m ${TARGET_NAME} 
```
above this line

Reply:
Yes, that should be fine.

Reply:
Done